components/policies/development/kueue/pre-sync-hook.yaml

```diff
@@ -0,0 +1,104 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kueue-crd-checker
+  annotations:
+    argocd.argoproj.io/hook: PreSync
+    argocd.argoproj.io/hook-delete-policy: BeforeHookCreation
```

Comment:
Should we use `HookSucceeded` instead? It seems to me that with `BeforeHookCreation` this resources will always be around, just updated before the next wave. I'd rather prefer them to be gone if everything applied correctly

https://argo-cd.readthedocs.io/en/stable/user-guide/sync-waves/#hook-lifecycle-and-cleanup

Reply:
thanks for the explanation